src/style.css

```diff
@@ -0,0 +1,90 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 403px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto, sans-serif;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+  padding: 14px 14px 15px;
+}
+
+.product-image {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 0;
+}
+
+.product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 45px auto 0;
+}
+
+.product-code {
+  margin: 5px 0 0;
+}
+
+.feedbacks-line {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.average-rate {
+  display: flex;
+  align-items: center;
+}
+
+.star:not(:last-child) {
+  margin-right: 6px;
+}
+
+.price-line {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+  margin-top: 21px;
+}
+
+.price {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.average-rate,
+.feedbacks,
+.price-text,
+.price {
+  margin: 0;
+}
+
+.order-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 170px;
+  height: 40px;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  text-transform: uppercase;
+  text-decoration: none;
+  background: #00acdc;
+  border-radius: 5px;
+  margin-top: 20px;
```

Comment:
`cursor: pointer;`

Reply:
it's default value for link, am I wrong?